src/core/utils/AutoId/AutoId.tsx

```diff
@@ -5,7 +5,7 @@
  * Idea of the AutoId is to patch incremented id after the first render.
```

Comment:
Would it be nice to have index in the `utils` level that when using utils, no need to tell the directories so much?

From current 
`import { AutoId } from '../../../utils/AutoId/AutoId';`
 to 
`import { AutoId } from '../../../utils';`

Reply:
Might make sense

Reply:
Tried this, but it makes imports difficult to read due to relative paths with multiple utils folders. Decided not to implement this with this PR.